assets/styles/common/_variables.scss

```diff
@@ -134,7 +134,7 @@ $tag_orange-light: #fee3dc;
 $tag_red: #dc5d5d;
 $tag_red-light: #f6d6d6;
 
-$box-shadow: 0 0 16px 0 rgba(0, 0, 0, 0.08);
+$box-shadow: 0 0.938em 4.125em -0.938em rgba(9, 0, 44, 0.12);
```

Comment:
This is used across the web, beware of changes, especially EM units if some element has already defined font-size!

Reply:
Shadow should be changed on whole site (change was requested by @laci-qualityunit).
Unit was changed to px.